DemoApp/Sources/Views/Login/LoginView.swift

```diff
@@ -53,6 +53,17 @@ struct LoginView: View {
                     } icon: {
                         Image(systemName: "person.crop.circle.badge.clock.fill")
                     }
+                    
+                    LoginItemView {
```

Comment:
Should we hide this one in Release builds?

Reply:
Same as above.